presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -2303,14 +2304,16 @@ private static ConnectorPushdownFilterResult pushdownFilter(
             RowExpression filter,
             Optional<ConnectorTableLayoutHandle> currentLayoutHandle)
     {
+        HiveTransactionManager hiveTransactionManager = new HiveTransactionManager();
         SubfieldExtractionRewriter filterPushdown = new SubfieldExtractionRewriter(
                 session,
                 new PlanNodeIdAllocator(),
                 rowExpressionService,
                 functionResolution,
                 functionMetadataManager,
-                new HiveTransactionManager(),
-                partitionManager);
+                hiveTransactionManager,
+                partitionManager,
+                t -> HiveFilterPushdown.getConnectorMetadata(hiveTransactionManager, t));
```

Comment:
```suggestion
                tableHandle -> HiveFilterPushdown.getConnectorMetadata(hiveTransactionManager, tableHandle));
```

Reply:
tableHandle is already defined on line 2303: https://github.com/prestodb/presto/pull/21574#diff-97caffd1bf83a7671054a8dbd37f2d7a7aebeba74d2cc02f976b4adbb2cce8aeR2303
![Screenshot 2023-12-20 at 11 03 41 AM](https://github.com/prestodb/presto/assets/64175/64be2636-603f-4b2c-b1c2-90fab380c3c5)
that's why I used a different name. Do we have a naming convention for the lambda function? tableHandleForLambda? tH? or `tableHandle_`?

Reply:
just rename `ConnectorTableHandle tableHandle` var as `ConnectorTableHandle connectorTableHandle`
and keep the lambda as tableHandle

Reply:
@ajaygeorge @tdcmeehan I already updated the variable naming and also make the commit message more clear.